app/helpers/publishers_helper.rb

```diff
@@ -59,25 +59,27 @@ def publisher_overall_bat_balance(publisher)
   def publisher_converted_overall_balance(publisher)
     return if publisher.uphold_connection.default_currency == "BAT" || publisher.uphold_connection.default_currency.blank?
 
-    publisher = publisher&.become_subclass
-
-    if publisher.partner?
-      balance = publisher.balance_in_currency
-    elsif publisher.only_user_funds?
-      balance = publisher.wallet&.contribution_balance&.amount_default_currency
-    elsif publisher.no_grants?
-      amount =  publisher.wallet&.overall_balance&.amount_default_currency - publisher.wallet&.contribution_balance&.amount_default_currency
-    else
-      balance = publisher.wallet&.overall_balance&.amount_default_currency
-    end
+    result = I18n.t("helpers.publisher.conversion_unavailable", code: publisher.uphold_connection.default_currency)
+    sentry_catcher do
+      publisher = publisher&.become_subclass
 
-    if balance.present?
-      I18n.t("helpers.publisher.balance_pending_approximate",
-             amount: '%.2f' % balance,
-             code: publisher&.uphold_connection.default_currency)
-    else
-      I18n.t("helpers.publisher.conversion_unavailable", code: publisher.uphold_connection.default_currency)
+      if publisher.partner?
+        balance = publisher.balance_in_currency
+      elsif publisher.only_user_funds?
+        balance = publisher.wallet&.contribution_balance&.amount_default_currency
+      elsif publisher.no_grants?
+        balance =  publisher.wallet&.overall_balance&.amount_default_currency - publisher.wallet&.contribution_balance&.amount_default_currency
```

Comment:
It seems like this add a lot of extra additions to Sentry when we know that the error is that something is going on with the Balance class.

Perhaps it would be better to return `0` if the `amount_default_currency` is nil